doc/Extensions/Applications.md

```diff
@@ -1629,52 +1629,56 @@ hide_monitoring_account = With this Boolean you can hide the Account which you
 
 ## SNMP Extend
 
-1: Copy the Perl script, smart, to the desired host. `wget
-https://github.com/librenms/librenms-agent/raw/master/snmp/smart -O
-/etc/snmp/smart`
+1. Copy the Perl script, smart, to the desired host.
 
-2: Run `chmod +x /etc/snmp/smart`
+```wget https://github.com/librenms/librenms-agent/raw/master/snmp/smart -O /etc/snmp/smart```
 
-3: Edit your snmpd.conf file and add:
+2. Run `chmod +x /etc/snmp/smart`
+
+3. Edit your snmpd.conf file and add:
 
 ```
 extend smart /etc/snmp/smart
 ```
 
-4: You will also need to create the config file, which defaults to the
-same path as the script, but with .config appended. So if the script
-is located at /etc/snmp/smart, the config file will be
-/etc/snmp/smart.config. Alternatively you can also specific a config
-via -c.
+4. You will also need to create the config file, which defaults to the same path as the script,
+but with .config appended. So if the script is located at /etc/snmp/smart, the config file
+will be `/etc/snmp/smart.config`. Alternatively you can also specific a config via `-c`.
 
-Anything starting with a # is comment. The format for variables is
-$variable=$value. Empty lines are ignored. Spaces and tabes at either
-the start or end of a line are ignored. Any line with out a = or # are
-treated as a disk.
+Anything starting with a # is comment. The format for variables is $variable=$value. Empty
+lines are ignored. Spaces and tabes at either the start or end of a line are ignored. Any
+line with out a matched variable or # are treated as a disk.
 
 ```
 #This is a comment
 cache=/var/cache/smart
-smartctl=/usr/bin/env smartctl
-useSN=1
+smartctl=/usr/local/sbin/smartctl
+useSN=0
 ada0
-ada1
+da5 /dev/da5 -d sat
```

Comment:
not really agree with your changes.
please let Options on default values in example
Also keep `ada1` as second example for direct device access.

Reply:
What helped me the most was actually:
```
A disk line is can be as simple as just a disk name under /dev/. Such as in the config above
The line "ada0" would resolve to "/dev/ada0" and would be called with no special argument. If
a line has a space in it, everything before the space is treated as the disk name and is what
used for reporting and everything after that is used as the argument to be passed to smartctl.
```

... which describe the format very clearly. Example lines aren't clear on their own, pretty opaque to say the least.

Anyway this is copypasted from the script itself, don't you think it should match?

Reply:
if you update Documentation, it's very welcome.
Examples should match default values.
It's correct the example in agent script should be fixed also.

Netherless the second example device 'ada1' should stay

Reply:
Okay, will do.

> Netherless the second example device 'ada1' should stay

What information does `ada1` add exactly? As `ada0` is already a "direct device access" example in the same format.

Reply:
ada1 is an second disc, for seeing how the next disc has to be added